enterprise/cluster/src/main/java/org/neo4j/cluster/protocol/atomicbroadcast/multipaxos/ProposerMessage.java

```diff
@@ -61,9 +64,22 @@ public Object getValue()
         @Override
         public String toString()
         {
+            Object toStringValue = value;
+            if (toStringValue instanceof Payload )
+            {
+                try
+                {
+                    toStringValue = new AtomicBroadcastSerializer( new ObjectStreamFactory(), new ObjectStreamFactory() ).receive( (Payload) toStringValue);
+                }
+                catch ( Throwable e )
+                {
+                    e.printStackTrace();
```

Comment:
Can we make way how we handle exceptions during `toString()` method call consistent? Now its either empty string either stack trace or nothing at all.